R/scATAC.Peak.annotation.R

```diff
@@ -641,21 +703,32 @@ peaks_closest_gene <- function(peaks, annotations=NULL, gene_element=NULL, TSSmo
   rownames(peaks_close) <- peaks_nam_not_annotated
   cat(dim(peaks_close), "\n")
   if (fast == F) {
-  if (TSSmode) {
-    cat(dim(peaks_annotated_TSS_ol), "\n")
-    cat(dim(peaks_annotated_TSS_on), "\n")
-    peaks_on_gene <- rbind(peaks_annotated_TSS_ol, peaks_annotated_TSS_on)
-    peaks_on_gene <- rbind(peaks_on_gene, peaks_annotated_end)
-    cat(dim(peaks_on_gene), "\n")
-    peaks_close <- rbind(peaks_on_gene, peaks_close)
-    cat(dim(peaks_close), "\n")
-  }
-  else{
-    cat(dim(peaks_annotated), "\n")
-    cat(dim(peaks_close), "\n")
-    peaks_close <- rbind(peaks_annotated, peaks_close)}
+    if (TSSmode) {
+      cat(dim(peaks_annotated_TSS_ol), "\n")
+      cat(dim(peaks_annotated_TSS_on), "\n")
+      peaks_on_gene <- rbind(peaks_annotated_TSS_ol, peaks_annotated_TSS_on)
+      peaks_on_gene <- rbind(peaks_on_gene, peaks_annotated_end)
+      cat(dim(peaks_on_gene), "\n")
+      peaks_close <- rbind(peaks_on_gene, peaks_close)
+      cat(dim(peaks_close), "\n")
+    }
+    else{
+      cat(dim(peaks_annotated), "\n")
+      cat(dim(peaks_close), "\n")
+      peaks_close <- rbind(peaks_annotated, peaks_close)}
+    }
+  else {
```

Comment:
I consider safer to only rename columns when `fast == T` because columns should have proper names otherwise. In this block: https://github.com/sposadac/scATAC.Peak.annotation/blob/fc34851748b35cf4c56b8170cdb706bdec213a57/R/scATAC.Peak.annotation.R#L563, you can see that the columns are named. Moreover, one have to handle the case for both TSSmode's. What do you think?